examples/compute/main.rs

```diff
@@ -0,0 +1,73 @@
+use dotrix::assets::Shader;
+use dotrix::camera;
+use dotrix::prelude::*;
+use dotrix::renderer::{
+    BindGroup, Binding, PipelineLayout, PipelineOptions, Stage, StorageBuffer, UniformBuffer,
+};
+use dotrix::{Assets, Camera, Pipeline, Renderer, World};
+
+const PARTICLES_COUNT: usize = 10;
+
+struct Particle {
+    params: UniformBuffer,
+    transforms: StorageBuffer,
+}
+
+fn main() {
+    Dotrix::application("Dotrix: Compute Example")
+        .with(System::from(startup))
+        .with(System::from(compute))
+        .with(System::from(render))
+        .run();
+}
+
+fn startup(
+    mut camera: Mut<Camera>,
+    mut world: Mut<World>,
+    mut assets: Mut<Assets>,
+    renderer: Const<Renderer>,
+) {
+    camera.target = [0., 0., 0.].into();
+    camera.distance = 2.0;
+    camera.xz_angle = 0.0;
+}
+
+fn compute(assets: Const<Assets>, world: Const<World>, mut renderer: Mut<Renderer>) {
```

Comment:
@QuantumEntangledAndy Here is mockup for comput API

Reply:
Seems ok but I'm not an expert on computer shaders so I'm not sure what we should include until it's fielded and we can see how it operates.